content/2017-munich/schedule.md

```diff
@@ -4,6 +4,9 @@ title: Schedule
 
 ## PromCon 2017 Schedule
 
+You can also access this schedule in a
+[mobile-friendly format](/2017-munich/mobile_schedule).
```

Comment:
Good point. Fixed (I just used "Android" instead of "mobile", as giggity is not useful for iphone users)